tests/test_lane.py

```diff
@@ -1257,13 +1257,24 @@ def test_search(self):
         eq_(wl.genre_ids, kw['in_any_of_these_genres'])
         eq_(1, kw['size'])
         eq_(0, kw['offset'])
-        
+
         # The single search result was converted to a MaterializedWorkWithGenre.
         [result] = results
         from model import MaterializedWorkWithGenre as mwg
         assert isinstance(result, mwg)
         eq_(work.id, result.works_id)
 
+        # Test that language and media are passed in
+        languages = ["fr"]
```

Comment:
It doesn't matter for purposes of this test, but we use the 3-letter language codes, so in a real scenario this would be 'fre'.